packages/core/src/modules/credentials/formats/CredentialFormatService.ts

```diff
@@ -88,4 +90,12 @@ export abstract class CredentialFormatService<CF extends CredentialFormat = Cred
 
     return attachment
   }
+
+  public areCredentialsEqual = (message1: Attachment, message2: Attachment): boolean => {
```

Comment:
I don't think this needs to be public, and is something specific to the jsonld credential format service. Can we make this a private method of that class?

Also is the FIXME resolved?

Reply:
Yes, it uses the new deepEqual method

Reply:
The method is public because it is accessed from the FormatService test. Is there a better way to allow that access?

Reply:
I think you can test it as a black box, through another method. So if the method that calls it receives false, it should trhow an error or something. I avoid making methods public for tests, as that now means it's part of the api

Reply:
As mentiond, this is only needed for the JsonLD Credential Format Service, can we move it to that class?